releasenotes/notes/fix-istioctl-describe-ingressinfo.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: release-notes/v2
+
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents.
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix.
+# - security-fix -- Used to specify that this change represents a vulnerability fix.
+# - feature -- Used to specify a new feature that has been added.
+# - test -- Used to describe additional testing added. This file is optional for
+#   tests, but included for completeness.
+kind: bug-fix
+
+# area describes the area that this change affects.
+# Valid values are:
+# - traffic-management
+# - security
+# - telemetry
+# - installation
+# - istioctl
+# - documentation
+area: istioctl
+
```

Comment:
Can you provide a link to the [issue ](https://github.com/istio/istio/issues/46160)this resolves? Similar to this note [here](https://github.com/istio/istio/blob/ea6008b534c83a814c3fc9c9322124be429d62e0/releasenotes/notes/49965.yaml)

Reply:
yep

Reply:
PTAL 😄